examples/first-frame/main.cpp

```diff
@@ -112,42 +112,46 @@ int main(int argc, char *argv[]) {
     command.parseArguments(argc, argv, command_map);
     int result = command.checkArgumentExist(command_map, arg_error);
     if (result != 0) {
-        std::cout << "Argument " << arg_error << " doesn't exist."
-                  << "\n"
-                  << Help_Menu;
+        LOG(ERROR) << "Argument " << arg_error << " doesn't exist! "
```

Comment:
It's a general practice to show the help menu whenever the user doesn't provide the right arguments to the command line.

Reply:
Changed all help menu printing to be done with cout